examples/kotlin/benchmarks/src/TestInheritedBenchmark.kt

```diff
@@ -0,0 +1,34 @@
+package test
+
+import org.openjdk.jmh.annotations.*
+import java.util.concurrent.TimeUnit
+import kotlin.math.cos
+import kotlin.math.sqrt
+
+@State(Scope.Benchmark)
+@Fork(1)
+@Warmup(iterations = 0)
+@Measurement(iterations = 1, time = 1, timeUnit = TimeUnit.SECONDS)
+abstract class BaseBenchmark {
+
+    protected lateinit var data: TestData
+
+    @Setup
+    fun setUp() {
+        data = TestData(50.0)
+    }
+}
+
+class TestInheritedBenchmark: BaseBenchmark() {
```

Comment:
Can a `TestInheritedBenchmark` subclass declare benchmark methods as well?

Reply:
Could you expand a bit on your question?

Reply:
I am referring to cases of multi-hierarchy benchmark classes, within which multiple classes define benchmark methods.